src/layout/main/contacts.scss

```diff
@@ -0,0 +1,243 @@
+        color: #161616;
+
+        @media only screen and (min-width: 744px) and (max-width: 1024px) {
+          &:first-child {
+            font-family: Cinzel;
+            font-size: 16px;
+            font-weight: 700;
+            line-height: 27px;
+            text-align: left;
+
+          }
+
+        }
+      }
+
+      &.card__description {
+
+        &_phone {
+          margin-top: 24px;
+
+          font-family: Cinzel;
+          font-size: 14px;
+          font-weight: 400;
+          line-height: 27px;
+        }
+
+        &_number {
+          color: #3E3E3E;
+          font-family: Cinzel;
+          margin-top: 4px;
+          font-size: 16px;
+          font-weight: 700;
+          line-height: 27px;
+          text-align: left;
+
+        }
+
+        &_address {
+          color: #161616;
+          margin-top: 8px;
+          font-size: 16px;
+          font-weight: 700;
+          line-height: 27px;
+          text-align: left;
+        }
+
+        &_street {
+          color: #3E3E3E;
+          margin-top: 4px;
+          font-family: Cinzel;
+          font-size: 16px;
+          font-weight: 700;
+          line-height: 22.4px;
+          text-align: left;
+
+        }
+      }
+    }
+  }
+
+  &__form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+
+    @media only screen and (min-width: 320px) and (max-width:744px) {
+      margin-top: 48px;
+      margin-left: 0;
+    }
+
+    @media only screen and (min-width: 744px) and (max-width: 1024px) {
+      &:first-child {
+        margin-left: 75px;
+
+      }
+
+    }
+
+    margin-left: 111px;
+
+    &_input {
+      @media only screen and (min-width: 320px) and (max-width: 744px) {
+        width: 100%;
+      }
+
+      @media only screen and (min-width: 744px) and (max-width: 1024px) {
+        font-family: Cinzel;
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 20px;
+        text-align: left;
+        padding: 11px 14px;
+
+
+
+      }
+
+      margin-top: 16px;
+      width: 411px;
+      padding: 16px 14px;
+      color: #7E7E83;
+      font-family: Cinzel;
+      font-size: 14px;
+      font-weight: 400;
+      line-height: 20px;
+      text-align: left;
+      border: none;
+      background-color: #F8F8FA;
+
+      &:first-child {
+        margin-top: 0;
+      }
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    &.textarea {
+      margin: 0;
+
+      height: 144px;
+
+      color: #7E7E83;
+      font-family: Cinzel;
+    }
+
+    &_button {
+      transition: all 0.3s;
+      width: 100%;
+      padding: 15px 0;
+      margin-top: 32px;
+      font-family: Cinzel;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 27px;
+      text-align: center;
+      color: #fff;
+      border: none;
+      background-color: #D12D35;
+      cursor: pointer;
+
+
+      @media only screen and (min-width: 744px) and (max-width: 1024px) {
```

Comment:
One great SCSS feature and advantage over simple CSS is the usage of Mixins.
This @media queries could be changed to @mixins instead, so you can avoid this long line repetition and make it more understandable.
For example:
```
@mixin on-tablet {
  @media only screen and (min-width: 744px) and (max-width: 1024px) {
    @content;
  }
}
```

Then, you can simply apply it wherever you need:
```
&_button {
  ...<button styling here>

  @include on-tablet {
      ...<media query conditional styling goes here>
   }
}
```

Reply:
And this is valid for all media queries